pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -364,7 +364,7 @@ def make_package(args):
         with open(join(dirname(__file__), 'dist_info.json'), 'r') as dist_info:
             dist_data = json.load(dist_info)
         arch = dist_data["archs"][0]
-        arch_dict = {"arm64-v8a": "8", "armeabi-v7a": "7", "x86": "6"}
+        arch_dict = {"x86_64": "9", "arm64-v8a": "8", "armeabi-v7a": "7", "x86": "6"}
         arch_code = arch_dict[arch]
```

Comment:
Would you mind changing this line to `arch_code = arch_dict.get(arch, '5')`, just to avoid issues if the user's arch is not known?

Reply:
well spotted. maybe even default to "1" which gives more space for any other unforeseen reworks?